.changeset/big-actors-tap.md

```diff
@@ -0,0 +1,6 @@
+---
+"@baseapp-frontend/authentication": major
+"@baseapp-frontend/utils": major
```

Comment:
I agree with the major bump, it would be nice if we could have a separate changelog for each one so we can guide people on what she/he should pay attention to when upgrading from a previous version

Reply:
Ok yeah I agree, I'll make sure that we have a changelog per package